examples/bigquery-auditlog-anomaly-detection/requirements.txt

```diff
@@ -0,0 +1,16 @@
+voila
```

Comment:
Please pin each dependency to the version needed for this to run. It's a good practice

Reply:
Done!